common/council_positions/00_council_positions.txt

```diff
@@ -208,6 +208,342 @@ councillor_court_chaplain = {
 	
 	name = {
 		first_valid = {
+			triggered_desc = {
+				trigger = {
+					scope:councillor_liege.faith = faith:catholic
+				}
+				desc = {
+					first_valid = {
+						triggered_desc = {
+							trigger = {
+								scope:councillor_liege.culture_group = culture_group:goidelic_group
+								scope:councillor_liege.highest_held_title_tier = tier_county
+							}
+							desc = count_male_theo_cath_goidelic
```

Comment:
Naming conventions here are off? Do (like here) or don't (like 7 lines later) we mark "theo"?